nemoguardrails/actions/llm/utils.py

```diff
@@ -574,4 +576,6 @@ def escape_flow_name(name: str) -> str:
         .replace('"', "")
         .replace("-", "_")
     )
-    return re.sub(r"\b\d+\b", lambda match: f"_{match.group()}_", result)
+    result = re.sub(r"\b\d+\b", lambda match: f"_{match.group()}_", result)
+    result = re.sub(r"\b\d*|[^\w\s]", "", result)
```

Comment:
I think you can update this to:
`result = re.sub(r"\b\d+|[^\w\s]", "", result)`
to match fewer times.

Reply:
Yes, makes sense to me. The current regex is matching every word bound.